### PR TITLE
Add Arm DevSummit 2022

### PR DESCRIPTION
**Pull request by bulislaw**
Add Arm DevSummit 2022 to the list.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,6 +84,7 @@ This tech conferences agenda list can be seen in https://developers.events/
 * 25-27: [SREcon22 Europe/Middle East/Africa](https://www.usenix.org/srecon) - Amsterdam (Netherlands)
 * 26-27: [DevOpsDays Montreal](https://devopsdays.org/) - Montreal (Canada) <a href="https://docs.google.com/forms/d/e/1FAIpQLSf4qwC-lcdymPpvd15hUFqR6lVDABvXXcFNZrIwvNeugG7MgQ/viewform"><img alt="CFP DevOpsDays Montreal" src="https://img.shields.io/static/v1?label=CFP&message=until%2030-April-2022&color=red"> </a>
 * 26-27: [API Days London](https://www.apidays.global/london/) - London (UK) & Online
+* 26-27: [Arm DevSummit 2022](https://devsummit.arm.com/?utm_source=armemployee&utm_medium=github&utm_%20campaign=2022_armdevsummit_mk17_arm_na_na_liveandondemand) - San Francisco (USA)
```
Comment (scraly) [inline README.md:87]:
```suggestion
* 26-27: [Arm DevSummit 2022](https://devsummit.arm.com/) - San Francisco (USA)
```

**Review by scraly — CHANGES_REQUESTED** [review]
thanks :) 
can you fix the link in order to remove marketing parameters? :)

**Comment by bulislaw** [comment]
Done :)

**Comment by scraly** [comment]
thanks :)

**Pull request merged**